docs/src/Layer.doc.js

```diff
@@ -44,27 +44,16 @@ card(
   "
     name="Overlaying Content"
     defaultCode={`
-class Example extends React.Component {
-  constructor(props) {
-    super(props);
-    this.handleToggle = this._handleToggle.bind(this);
-    this.state = {
-      showLayer: false,
-    };
-  }
+function Example() {
 
-  _handleToggle() {
-    this.setState(prevState => ({ showLayer: !prevState.showLayer }));
-  }
+  const [showLayer, setShowLayer] = React.useState(false);
 
-  render() {
-    const { showLayer } = this.state;
     return (
```

Comment:
@AlbertCarreras The `return` doesn't look indented correctly (remove spaces before it).

Before
![image](https://user-images.githubusercontent.com/127199/80012461-38b19b00-8482-11ea-862c-097f6dbfc741.png)

After
![image](https://user-images.githubusercontent.com/127199/80012487-3f401280-8482-11ea-98ae-022964b86392.png)

Reply:
DONE